showcase/tests/acceptance/percy-test.js

```diff
@@ -169,6 +169,9 @@ module('Acceptance | Percy test', function (hooks) {
     await visit('/components/text');
     await percySnapshot('Text');
 
+    await visit('/components/time');
+    await percySnapshot('Time');
```

Comment:
We made a decision to not include the Time component in these tests originally as it didn't really include defined visual styles. As this PR adds a more defined visual style now I wanted to add it but it seems to have wiped out the other snapshots. Is there something I missed or did wrong?

Reply:
I believe this to be a case where Percy failed, rather than something to do with the changes in this PR. Unfortunately, there's no way to retrigger Percy manually, so we'd either have to trust it does what we expect it to do, or push a new commit/rebase to force a new trigger.

Reply:
I can try rebasing once there's something new in `main`.

Reply:
Oh I just remembered another issue with visual testing of Time is that the times change which messes up the visual tests. Actually this could be causing the issue. I'll try scoping the test to a limited section and add a comment.

Reply:
Ah, I see you're already doing it in this PR

Reply:
I've tried to narrow the parts tested but it still triggers this error:
`Error: infinite rendering invalidation detected`

I think that unfortunately the Time component just can't be easily tested with Percy. Unless there's a way to force it to render once and then stop updating. I just don't have a good idea on a practical way to do it currently.